src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -146,7 +146,7 @@ class GuidancePage extends React.Component {
             Important tips for completing Form {form.num}:
           </p>
           {this.props.formValues['4_reason'] === '8' ? dd214Tips : nonDd2014Tips}
-          <a target="_blank" href={form.link} className="usa-button-primary va-button" ref={(el) => { this.downloadFormBtn = el; }}>Download Form {form.num}</a>
+          <a target="_blank" href={form.link} className="usa-button-primary va-button" ref={(el) => { this.downloadFormBtn = el; }}>Download Form {form.num}<span className="exit-icon"></span></a>
```

Comment:
I noticed `va-button` class is used in a few places in this app. It's not a class that is defined anywhere, and I think it was meant to be `usa-button`.